solana/programs/swap-layer/src/processor/initiate/transfer.rs

```diff
@@ -0,0 +1,194 @@
+use crate::{
+    composite::*,
+    error::SwapLayerError,
+    state::{Custodian, Peer, StagedOutbound, StagedRedeem},
+};
+use anchor_lang::prelude::*;
+use anchor_spl::token;
+use common::wormhole_io::{Readable, TypePrefixedPayload};
+use swap_layer_messages::{
+    messages::SwapMessageV1,
+    types::{OutputToken, RedeemMode, Uint48},
+};
+
+#[derive(Accounts)]
+pub struct InitiateTransferNew<'info> {
+    #[account(mut)]
+    /// The payer of the transaction. This could either be the recipient or a relayer.
```

Comment:
Don't think this can be a relayer? Is this a stale comment?

Reply:
Fixed here: https://github.com/wormholelabs-xyz/swap-layer/pull/30/commits/264000226f6942e5e698d1dcffc5cb1dc2bc033e